README.md

```diff
@@ -81,9 +81,13 @@ pip install -e homu
 
 1. Copy `cfg.sample.toml` to `cfg.toml`, and edit it accordingly.
 
-2. Create a GitHub account that will be used by Homu. You can also use the
-   existing account. In the account settings, register a new application and
-   generate a new access token.
+2. Create a GitHub account that will be used by Homu. You can also use
+   an existing account. In the [account settings][settings], register
+   a new application and generate a new access token. Permissions
+   required:
+    - `push`
+    - `issue_comment`
```

Comment:
Checking the account settings again, those permissions only exist in the per-repository settings. In fact, the application registration interface doesn't list any permissions at all. I guess those should be moved to the step 3. Also, they're called "events" in the per-repository settings, not "permissions".